tcpMaxConn.go

```diff
@@ -29,15 +33,35 @@ func runTcpConnectionsInParallel(numberConnections int, delay int, host string,
 }
 
 func main() {
-	hostPtr := flag.String("host", "localhost", "Host you want to open tcp connections against")
+	hostPtr := flag.String("host", "", "Host you want to open tcp connections against")
 	// according to https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers, you are probably not using this
 	portPtr := flag.Int("port", 9998, "Port you want to open tcp connections against")
 	numberConnectionsPtr := flag.Int("connections", 100, "Number of connections you want to open")
 	delayPtr := flag.Int("delay", 10, "Number of ms you want to sleep between each connection creation")
-
 	flag.Parse()
 
-	runTcpConnectionsInParallel(*numberConnectionsPtr, *delayPtr, *hostPtr, *portPtr)
+	if *hostPtr == "" {
+		flag.PrintDefaults()
+		os.Exit(1)
+	}
+
+	fmt.Println("*** WARNING ***")
+	fmt.Println("You are going to run a TCP stress check with the following arguments:")
+	fmt.Println("  - Target: " + *hostPtr)
+	fmt.Println("  - TCP Port: " + strconv.Itoa(*portPtr))
+	fmt.Println("  - # of concurrent connections: " + strconv.Itoa(*numberConnectionsPtr))
+
+	reader := bufio.NewReader(os.Stdin)
+	fmt.Println("Please, would you confirm this execution? (y/N)")
```

Comment:
why "y" and "N", so this difference in capitalization? i'd start and endless loop, that does not terminate unless the user writes :"yes", "y", "no", "n" (in any capitalization form)

Reply:
N is the default if you press anything different from y